include/classes/Language/LanguageEditor.class.php

```diff
@@ -119,11 +115,11 @@ function checkFilter($param) {
 
 	// public
 	function printTerms($terms){
-		global $addslashes, $languageManager; // why won't $addslashes = $this->addslashes; work?
```

Comment:
To answer your question, i believe it's because Language class is not extended from DAO class at the moment.